hydra.yaml

```diff
@@ -0,0 +1,576 @@
+
+    Project:
+      type: object
+      properties:
+        owner:
+          description: username of the project owner
+          type: string
+        name:
+          description: name of the project
+          type: string
+        displayname:
+          description: name to be displayed in the web interface
+          type: string
+        description:
+          description: description of the project
+          type: string
+        hidden:
+          description: when set to true the project is not displayed in the web interface
+          type: boolean
+        enabled:
+          description: when set to true the project gets scheduled for evaluation
+          type: boolean
+        jobsets:
+          description: list of jobsets belonging to this project
+          type: array
+          items:
+            type: string
+        releases:
+          description: list of releases of this project
+          type: array
+          items:
+            type: string
+
+    JobsetInput:
+      type: object
+      properties:
+        jobsetinputalts:
+          type: array
+          description: ???
+          items:
+            type: string
+
+    Jobset:
+      type: object
+      properties:
+        fetcherrormsg:
+          nullable: true
+          description: contains the error message when there was a problem fetching sources for a jobset
+          type: string
+        nixexprinput:
+          description: the name of the jobset input which contains the nixexprpath
+          type: string
+        errormsg:
+          description: contains the stderr output of the nix-instantiate command 
+          type: string
+        emailoverride:
+          description: email address to send notices to instead of the package maintainer (can be a comma separated list)
+          type: string
+        nixexprpath:
+          nullable: true
+          description: the path to the file to evaluate
+          type: string
+        enabled:
+          description: when set to true the jobset gets scheduled for evaluation
+          type: boolean
+        jobsetinputs:
+          description: inputs configured for this jobset
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/JobsetInput'
+
+    JobsetEvalInput:
+      type: object
+      properties:
+        uri:
+          nullable: true
+          description: URI of this input (which differs depending on the type of the input)
+          type: string
+        'type':
+          description: The type of this input # XXX: should be an enum
+          type: string
+        revision:
+          nullable: true
+          description: ???
+          type: string
+        value:
+          nullable: true
+          description: ???
+          type: boolean # ?? seems to be either 'false' or null ?
+        dependency:
+          nullable: true
+          description: ???
+          type: string # ?? i have no idea this is always null
```

Comment:
I think this is the build ID of another build. Probably another misfeature that we shouldn't document (since using builds as inputs is not very reproducible).

Reply:
i prefixed the description with a big **DEPRECATED** and I will remove it as soon as this isn't in the code anymore. 

I don't want to leave it out completely: I don't want `hydra-api.yaml` to have the same fate as `doc/manual/api.xml` meaning i want to verify that the specification is correct in CI. Not as a first step, but eventually. Hence my desire to stick to what is actually there (the same also applies for `jobsetinputalts` which are also still there.